internal/common/db.go

```diff
@@ -6,4 +6,8 @@ type Cluster struct {
 	models.Cluster
 	// The pull secret that obtained from the Pull Secret page on the Red Hat OpenShift Cluster Manager site.
 	PullSecret string `json:"pull_secret" gorm:"type:TEXT"`
+
+	// The compute hash value of the http-proxy, https-proxy and no-proxy attributes, used internally to indicate
+	// if the proxy settings were changed while downloading ISO
+	ProxyHash string
```

Comment:
Did you forget to commit the swagger change?  I would move ProxyHash to be an image property.

Reply:
There are no swagger changes. The purpose was to add it to the DB level only so this property will remain internal and won't be exposed to the user / client. 

For that purpose it was added specifically to the common.Cluster since there is no equivalent of 'common.Image' in which it can be added.

Pls let me know if you prefer it to be part of swagger / client and I'll then move it to the Image property.